test/lib/rules/ssr/no-unsupported-node-api.js

```diff
@@ -18,6 +18,16 @@ testRule('ssr/no-unsupported-node-api', {
                 }
             `,
         },
+        {
+            code: `
+            if (!import.meta.env.SSR) {
+                const fs = require('node:fs');
+
+                // Unsupported Node API call with node: prefix
```

Comment:
Is this comment still relevant?

Reply:
I put it there to highlight that usage of node api under ssr env skip check is still valid

Reply:
In this case the API is supported? I think the comment might be confusing.

Reply:
Okay I will remove that